sdk/web-pubsub/web-pubsub/samples-dev/managingGroups.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates adding and removing users from groups
+ */
+
+import { WebPubSubServiceClient } from "@azure/web-pubsub";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const chatHub = new WebPubSubServiceClient(process.env.WPS_CONNECTION_STRING!, "chat");
+const adminGroup = chatHub.group("admin");
+
+async function main() {
+  // adding and removing users
+  await adminGroup.addUser("bterlson");
+  await adminGroup.hasUser("bterlson"); // true
+  await adminGroup.removeUser("xirzec");
```

Comment:
Does removing a user that doesn't exist throw an error, or silently pass?

Reply:
It does not throw.